api/lib/apollo.js

```diff
@@ -28,6 +36,17 @@ function startBlockTriggers() {
   networks.map((network) => network.initialize())
 }
 
+async function checkIsValidIdToken(idToken) {
+  try {
+    const decodedToken = await firebaseAdmin.auth().verifyIdToken(idToken)
+    return true
```

Comment:
can we return the UID of the user if the token is correct?

Reply:
sure :ok_hand: